### ISSUE TITLE
Default page

### ISSUE THREAD

**Issue by overload08** [domogik/domoweb#100]
Une idée intéressante, pouvoir définir une section par défaut, via cookies pour en avoir une par équipement dans le cas il y aurait plusieurs tablettes dans la maison

**Comment by fritz-smh** [comment]
ça existe, c'est non documenté par contre à ce jour :)

Dans l'url, faire http://ip:port/<id> avec l'id de la session (le chiffre qu'on retrouve à côté du nom de la section dans la page de config de la section)

Ou http://ip:port/<nom de la section>

Dis moi si c'est ok pour toi

**Comment by overload08** [comment]
Yes! C'est tout bon (Petit coquin va, c'est pas bien de cacher des choses comme ça!)

**Comment by fritz-smh** [comment]
Je rouvre pour penser à compléter la doc ;)
